modules/complex-number/test/test_shagov_maksim_complex_number.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 Maksim Shagov
+
+#include <gtest/gtest.h>
+
+#include <tuple>
+#include "include/complex_number.h"
+
+TEST(Shagov_Maksim_ComplexNumberTEST, Constructor) {
+    ComplexNumber number;
+    ASSERT_EQ(number.getRe(), 0);
+    ASSERT_EQ(number.getIm(), 0);
+}
+
+TEST(Shagov_Maksim_ComplexNumberTEST, Operator_equals) {
+    ComplexNumber number(13.7, -11.2);
+    ComplexNumber number1;
+
+    number1 = number;
```

Comment:
Возможно, лучше сразу сделать присваивание?
```suggestion
    ComplexNumber number1 = number;
```